araproc/framework/dataset.py

```diff
@@ -18,14 +18,29 @@
 def file_is_safe(file_path):
     
     if not isinstance(file_path, str):
-        raise TypeError("Path to file must be a string")
+        raise TypeError(f"Path to file must be a string. Is {type(file_path)}")
     if not os.path.exists(file_path):
         raise FileNotFoundError(f"File ({file_path}) not found")
     if not os.path.isfile(file_path):
         raise ValueError(f"{file_path} looks like a directory, not a file")
     
     return True
 
+def get_cvmfs_ped_file_name(station_id, run_number):
+
+    if station_id not in const.valid_station_ids:
+        raise Exception(f"Station id {station_id} is not supported")
+
+    if not np.isfinite(run_number) and run_number > 0:
```

Comment:
I'm not sure I understand the second condition? Doesn't that allow negative run numbers to move on?

Reply:
Unless it's the order of the boolean operations I'm missing?

Reply:
Nope, that's just a plain typo. Fixed, thanks!